Apps/AdminWebClient/src/ClientApp/src/components/covidTreatmentAssessment/CovidTreatmentAssessment.vue

```diff
@@ -177,14 +193,53 @@ export default class CovidTreatmentAssessmentComponent extends Vue {
         }
     }
 
+    private setAddressRequest(): void {
+        this.covidTreatmentAssessmentRequest.country = this.address.country;
+        this.covidTreatmentAssessmentRequest.postalCode =
+            this.address.postalCode;
+        this.covidTreatmentAssessmentRequest.provOrState = this.address.state;
+        this.covidTreatmentAssessmentRequest.streetAddress =
+            this.address.streetLines;
+    }
```

Comment:
Setting the patient's name and PHN could be moved into this function if it was renamed to something like `populateRequest()`.

Reply:
@MikeLyttle  I did not use the word "request" because I am only setting partial fields of the request model. But will change it